### DISCUSSION TITLE
Struggling to find information on LaTeX sections -> Markdown directives

### DISCUSSION THREAD

**Discussion by dmca-glasgow** [jgm/pandoc#9449] [General]
Hi there, I'm new to Pandoc!

I'm struggling to find more information on how Pandoc converts LaTeX's `\begin{name} ... \end{name}` sections to Markdown "directives".

I have the following LaTeX:

```latex
\begin{example}
Test example
\end{example}

\begin{definition}
Test definition
\end{definition}

\begin{task}
Test task
  \begin{answer}
  Test answer
  \end{answer}
\end{task}
```

When running the command:

```sh
pandoc --output test.md test.tex
```

The resulting Markdown looks like this:

```md
::: example
Test example
:::

::: definition
Test definition
:::

::: task
Test task

::: answer
Test answer
:::
:::
```

Which is _almost_ ideal for my usecase, with one exception: my existing Markdown to HTML conversion tool uses [micromark-extension-directive](https://github.com/micromark/micromark-extension-directive) which has a different rule for nesting (search their readme for "nest" to see their example).  Its syntax requires outer directives to increment preceding colons, for example:

```md
:::: task
Test task

::: answer
Test answer
:::
::::
```

I had a look at creating a Lua filter for this, but I'm struggling to find the right Pandoc term for this structure.  

Also, I don't need to support the nesting logic at this stage, I think I can get away with a special case for `task`, for example:

```
if (name === 'task')
   output 4 colons
else
  output 3 colons
```

Any pointers? Thanks.

**Comment by jgm** [comment]
It wouldn't hurt for us to increase the length of the colons depending on nesting. (Our syntax doesn't require this, but it permits it.)  So that's probably a decent change to consider, especially if it aids compatibility.

**Reply by jgm** [reply]
Go ahead and submit an issue for this (targeting the markdown writer).

**Reply by dmca-glasgow** [reply]
Will do, thanks

**Reply by dmca-glasgow** [reply]
Done: https://github.com/jgm/pandoc/issues/9450
